plugins/funind/glob_termops.mli

```diff
@@ -122,3 +122,10 @@ val zeta_normalize : Glob_term.glob_constr -> Glob_term.glob_constr
 
 
 val expand_as : glob_constr -> glob_constr
+
+
+(* [resolve_and_replace_implicits ?expected_type env sigma rt] solves implicits of [rt] w.r.t. [env] and [sigma] and then replace them by their solution 
+ *)
+val resolve_and_replace_implicits :
```

Comment:
@herbelin I'm not sure to understand clearly if we can use that in `Function` or not since my first interest here is to keep a trace of the users terms (to prevent match explosion). We may have a quick live talk about this. 

If there is no more objection, I suggest that we merge this patch (maybe on 8.7 too).